apps/wallet-dashboard/components/transactions/TransactionsOverview.tsx

```diff
@@ -9,10 +9,7 @@ export function TransactionsOverview() {
     return (
         <Panel>
             <Title title="Activity" />
-            <div
-                className="h-full max-h-[400px] flex-1 overflow-y-auto px-sm pb-md  pt-sm sm:max-h-none"
-                data-testid="home-page-activity-section"
-            >
+            <div className="flex h-full w-full flex-col" data-testid="home-page-activity-section">
```

Comment:
this change affects the layout :face_with_peeking_eye: could you please revert it? 🙏